src/components/general-widgets/HTML/component.js

```diff
@@ -27,11 +27,17 @@ export class Component extends PureComponent {
 
     return children ? (
       <div>
-        <div dangerouslySetInnerHTML={{ __html: cleanHTMLString }} />
+        <div
+          className="html-widget"
```

Comment:
```suggestion
          className="html-container"
```

we use `-container` everywhere else